mkt/developers/utils.py

```diff
@@ -154,3 +154,14 @@ def escalate_prerelease_permissions(app, validation, version):
         escalate_app(
             app, version, nobody, 'App uses prerelease permissions',
             mkt.LOG.ESCALATION_PRERELEASE_APP)
+
+
+def prioritize_app(app, user):
+    if not app.priority_review:
```

Comment:
I was wanting to avoid multiple updates to the record in the database - `priority_review` is set with all the updates in that edit listing section in forms.py.  But the message and so on needs to go out if it was changed still.  Aware the code is a little weird though - I'm open to suggestions.